### ISSUE TITLE
A11y_mssql for VSCode_Results Tab_Screen Reader: NVDA/JAWS is not reading the total number of columns and rows correctly

### ISSUE THREAD

**Issue by Tanushagottipati** [microsoft/vscode-mssql#1227]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopApp; #Win10; #A11y_mssqlforVSCode; #MAS1.3.1; #C+E-H; #mssqlforVSCode; #ScreenReader; #NVDA; #JAWS

**Environment Details:**
Visual Studio Code – Insider (install from https://na01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fcode.visualstudio.com%2Finsiders.&data=02%7C01%7C%7C00d9f7de52714b323f9908d607afe0c5%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636704849297354687&sdata=cmMqckjoNPGtCS3SZ00bCqCnFprKgiux%2BIjyvzgU0hE%3D&reserved=0 )
Version: 1.34.0-insider (user Setup)
Mssql version: 1.6.0
OS Version: Microsoft Windows 10 Enterprise (10.0.18362 Build 18362)
Screen reader: NVDA, JAWS

**MAS Violated:** MAS 1.3.1

**Repro Steps:**
1.       Open Visual Studio Code in your system--
2.       Navigate to "Extensions"   and install "SQl Server (mssql)" extension
3.       Press F1 and select MS SqL :   Connect and connect to any sql server
4.       Write code (SELECT * FROM sys.objects;) in the new document and press F1 and select   ""MS SQL : Execute Query"" "  
5.       Start NVDA/JAWS   
6.        Tab till “more options” button and press down arrow in scan mode to read the non-text content   
7.         Verify if NVDA/JAWS is reading the total number of columns and rows correctly

**Actual Result:** 
When user presses down arrow while navigating to the grid, NVDA/JAWS is reading as table with 26 rows and 13 columns

**Expected Result:** 
Screen reader should read the total number of rows and columns correctly as “Table with 104 rows and 13 columns”

**User Impact:** 
Screen Reader users will not be aware of the total number of rows and columns that are present in the table   

**MAS Reference:** 
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}

![Bug_1227_NVDA is reading the number of rows wrong](https://user-images.githubusercontent.com/41109135/56639324-0eba7280-668e-11e9-87ce-2cdc13d5564d.png)

**Comment by kenvanhyning** [comment]
This should be fixed in 1.7, please verify and reopen if any issues are found.

**Comment by fsteffi** [comment]
#closed; 
logged new bug as issue still there https://github.com/Microsoft/vscode-mssql/issues/1227
